lldb/source/Utility/Status.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "lldb/Utility/Status.h"
 
+#include "lldb/Expression/DiagnosticManager.h"
```

Comment:
Is this addition necessary ? It doesn't look like anything else has been added to this file